.github/workflows/build.yml

```diff
@@ -164,10 +164,19 @@ jobs:
         id: install-oc
         if: ${{ steps.sanity_check_pr_content.outputs.report-exists != 'true' && steps.cache.outputs.cache-hit != 'true' && steps.check_build_required.outputs.run-build == 'true' }}
         run: |
+          echo "::set-output name=install-oc::true"
           curl -sLO https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz
           tar zxvf openshift-client-linux.tar.gz oc
           echo "::set-output name=oc-installed::true"
 
+      - name: Check oc installed
+        id: check_oc_installed
+        if: ${{ always() && steps.install-oc.outputs.install-oc == 'true' && steps.install-oc.outputs.oc-installed != 'true' }}
```

Comment:
I think it will work fine. As I am learning the github actions workflow, I am just wondering if we can use `steps.install-oc.conclusion == 'failure' ` here instead of comparing two outputs ` steps.install-oc.outputs.install-oc == 'true' && steps.install-oc.outputs.oc-installed != 'true' `

Reply:
thanks, actaully meant I could delete the new step entirely and just pass the step conclusion when generating the pr comments.